### ISSUE TITLE
example in pounds

### ISSUE THREAD

**Issue by r12a** [w3c/Micropub#38]
(Editorial nit) [by Addison Phillips]

https://www.w3.org/TR/micropub/#nested-microformats-objects

The example give at the location above is in pounds. For international accessibility, perhaps SI units should be used instead?

**Comment by aaronpk** [comment]
Thanks! Let me know if that looks good to you. https://github.com/w3c/Micropub/commit/df00b18d52f53d1bad814d0e18014987981d24fc

**Comment by r12a** [comment]
Addison agreed to close this
